_posts/2017-01-27-squad-health-checks.md

```diff
@@ -0,0 +1,70 @@
+
+In the first of our Agile Delivery Tech Talks last September, I did a presentation on the Agile Maturity Model that we developed. This was very much a deep-dive exercise intended to discover which areas required the most focus from both management and the squads themselves to ensure we were best meeting the needs of the business.
+
+Agile is a much used and abused term these days, but it's also something that we pride ourselves on as a business and as software delivery professionals. Many of the external communications used by our recruitment team focus on agility, we employ people based in part on their experience with agile methods and we even have coffee mugs saying "We bet you've never worked anywhere as agile".
+
+What we didn't have though was a shared understanding of exactly what was meant by this and more importantly, how to make sure we continued improving in a tangible way.
+
+We set about designing the model based on 4 high level themes:
+
+* **Foundations**, which represent agile hygiene factors such as squad size, process fundamentals and facilitation.
+* **Delivery Performance**, which encompasses our planning and forecasting capability, value focus and data-driven decision-making.
+* **Culture**, which deals with trust & respect, collaboration and teamwork and transparency.
+* **Leadership**, this concerns the presence and engagement of our technical and product leaders, including the maintenance of a well-ordered backlog and visible roadmap, with a shared vision and goals.
+
+Once we had these themes fleshed out into agile factors across 4 maturity levels, we had a very detailed view of the range of possibilities that our delivery squads could fall into. The next step was to run sessions to determine the scores (1-4) for each of these factors, which meant running through all 100 scenarios, to generate the scores.
+
+As you can imagine this was quite a time-consuming exercise and required around 1.5 hours to complete for each squad.
+
+### So what did we learn?
+We gained a great deal of insight from this exercise. Some squads determined that they didn't have as much insight into the future vision and goals as they wanted. Others felt that they didn't receive enough feedback as to the impact of the work that they had delivered. Some squads felt that they were too big, and may benefit from being separated into smaller sub-teams.
+
+We also learned that defining “agile” in itself is a big challenge, contains many overlaps between areas, and since it is such a wide-ranging umbrella term, a maturity model has to be very context-specific.
```

Comment:
"smart quotes" here around the term "agile", but regular quotes are used in the rest of the piece